packages/hardhat-core/test/internal/hardhat-network/provider/fork/ForkBlockchain.ts

```diff
@@ -66,6 +69,18 @@ describe("ForkBlockchain", () => {
     assert.instanceOf(fb, ForkBlockchain);
   });
 
+  // TODO: remove regex code
```

Comment:
What does this comment mean?

Reply:
Ugh, nothing, I forgot to cancel it. I'll remove it